README.md

```diff
@@ -1009,142 +1042,23 @@ This means that if you call `get` or `set` with a private field or accessor, the
-    - It is possible to write a `@frozen` class decorator which *mostly* works, but which prevents the use of static fields.
-- `@enumerable`: This decorator would make a method enumerable, overriding its default of non-enumerable. Decorators cannot change property attributes, as they do not receive property descriptors to manipulate them as in Stage 1 decorators, and they are not passed the constructor of the class to do so imperatively. This is to meet requirements from implementations that decorators leave classes with statically predictable shapes. Instead, changes like this could be done by `Object.defineProperty` calls after the class definition executes.
-- `@reader`: This decorator for a private field would create a public accessor to read it. It is impossible to create, as decorators are not given access to the class. Such a change in shape would run counter to the "static shape" goals from native implementers.
-
-## Open questions
-
-- **Accessor coalescing**: In the above proposal, getters and setters are decorated separately, whereas in earlier decorators proposals, they were coalesced into a unit which applies to the decorator together. This is done in order to keep the decorator desugaring simple and efficient, without the need for an intermediate data structure to associate getters with setters (which may be dynamic due to computed property names). Should decorator coalescing be restored?
-- **Metadata format**: How should metadata added by decorators be represented in the object graph? Should there be a built-in library of functions to query this metadata? How should adding metadata to class elements be timed relative to other observable operations with decorators?
-
 ## Standardization plan
 
-- Write spec text and tests and implement in experimental transpilers
-- Collect feedback from JavaScript developers testing the transpiler implementation
-- Iterate on open questions within the proposal, presenting them to TC39 and discussing further in the biweekly decorators calls, to bring a conclusion to committee in a future meeting
-- Propose for Stage 3 no sooner than six months after prototyping begins, so we have time to collect experience from developers in transpilers
+- [x] Iterate on open questions within the proposal, presenting them to TC39 and discussing further in the biweekly decorators calls, to bring a conclusion to committee in a future meeting
+  - STATUS: Open questions have been resolved, decorators working group has reached general consensus on the design.
+- [x] Write spec text
+  - STATUS: Complete, available [here](https://arai-a.github.io/ecma262-compare/?pr=2417).
+- [x] Implement in experimental transpilers
+  - STATUS: An experimental implementation has been created and is available for general use. Work is ongoing to implement in Babel and get more feedback.
+    - [x] Independent implentation: https://javascriptdecorators.org/
+    - [ ] Babel plugin implementation
```

Comment:
I'm interested in testing the transpiler in actual projects. 

While https://javascriptdecorators.org/ is great for getting a glance, is not doable to integrate in a project pipeline

Do you know if someone is working on babel implementation?

Last time i read something about, @nicolo-ribaudo said it would wait to proposal settle before implementing in babel

Reply:
As noted in the PR, the Babel plugin is in progress, and we intend to gather feedback from users of the plugin. I agree that the Babel plugin will give us much more feedback from many more users than the current implementation, since it will be part of a more standard build chain. I’ve been in touch with @nicolo-ribaudo, we’re both busy at the moment but have agreed to start working sometime later this summer.